docker/eradiate-jupyterlab/Dockerfile

```diff
@@ -1,12 +1,12 @@
-ARG REGISTRY=fxia
-ARG VERSION=v1.0.0
+ARG BASE_IMAGE
+ARG BASE_IMAGE_VERSION
 
-FROM ${REGISTRY}/eradiate:${VERSION}
+FROM ${BASE_IMAGE}:${BASE_IMAGE_VERSION}
 
 RUN pip3 install jupyterlab
```

Comment:
IIRC the dev Conda environment ships a configured Jupyter Lab; maybe you no longer need this?

Reply:
It was indeed superfluous. The pip install command has been removed